api/src/routes/certificate.test.ts

```diff
@@ -0,0 +1,336 @@
+import { Certification } from '../../../shared/config/certification-settings';
+import {
+  defaultUserEmail,
+  defaultUserId,
+  devLogin,
+  setupServer,
+  superRequest
+} from '../../jest.utils';
+import { SHOW_UPCOMING_CHANGES } from '../utils/env';
+
+describe('certificate routes', () => {
+  setupServer();
+  describe('Authenticated user', () => {
+    let setCookies: string[];
+
+    // Authenticate user
+    beforeAll(async () => {
+      setCookies = await devLogin();
+    });
+
+    afterEach(() => {
+      jest.restoreAllMocks();
+    });
+
+    describe('PUT /certificate/verify', () => {
+      beforeEach(async () => {
+        await fastifyTestInstance.prisma.user.updateMany({
+          where: { email: defaultUserEmail },
+          data: {
+            completedChallenges: [],
+            name: 'fcc',
+            isRespWebDesignCert: false,
+            isJsAlgoDataStructCert: false,
+            isFrontEndLibsCert: false,
+            is2018DataVisCert: false,
+            isRelationalDatabaseCertV8: false,
+            isApisMicroservicesCert: false,
+            isQaCertV7: false,
+            isSciCompPyCertV7: false,
+            isDataAnalysisPyCertV7: false,
+            isInfosecCertV7: false,
+            isMachineLearningPyCertV7: false,
+            isCollegeAlgebraPyCertV8: false,
+            isFoundationalCSharpCertV8: false,
+            username: 'fcc'
+          }
+        });
+      });
+
+      test('should return 500 if no certSlug', async () => {
+        const response = await superRequest('/certificate/verify', {
+          method: 'PUT',
+          setCookies
+        }).send({ certSlug: undefined });
```

Comment:
```suggestion
        }).send({});
```
Same thing, but shorter.